Anlab.Mvc/Controllers/AlertsController.cs

```diff
@@ -0,0 +1,130 @@
+using Anlab.Core.Data;
+using Anlab.Core.Domain;
+using AnlabMvc.Models.Roles;
+using Markdig;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AnlabMvc.Controllers
+{
+    [Authorize(Roles = RoleCodes.Admin)]
+    public class AlertsController : ApplicationController
+    {
+        private readonly ApplicationDbContext _dbContext;
+
+        public AlertsController(ApplicationDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+        public async Task<IActionResult> Index()
+        {
+            var alerts = await _dbContext.SystemAlerts.OrderBy(a => a.IsActive).ThenBy(a => a.Updated).ToListAsync();
+            return View(alerts);
+        }
+
+        // GET: AdminAnalysis/Create
+        public ActionResult Create()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> Create(SystemAlert systemAlert)

```

Comment:
Instead of making the Domain model have fake non-required fields and other stuff like empty created/updated, I think it'd just be simpler to create a viewModel here for this create method.  Just something that can get `IsActive`, markdown, description, etc the required form fields.  you don't have to use the domain SystemAlert for both DB and model binding